monitoring/init_pis.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+pswd="piman"
+hosts="../hosts.csv"
+
+
+while IFS= read -r line # Read hosts file one line at a time to get ip addresses of pis
+do
+    # Parse the line by replacing ; with spaces, then grabbing the ip address from the resulting array
+    base=${line##*/}
+    parsed_line="${line%/*}/${base//;/ }"
+    array=($parsed_line)
+    ip="${array[1]}"
+
+    # Now run necessary commands
+    echo "Setting up pi@$ip"
+    sshpass -p $pswd scp monitoring-server.py monitoring-pi.service monitoring-pi.sh monitoring-pi-installer.sh pi@$ip:
+    sshpass -p $pswd ssh -n pi@$ip ./monitoring-pi-installer.sh # Could run each pi in parallel with &
```

Comment:
use a path relative to $BASH_SOURCE rather than ./ i think something like $(basename "$BASH_SOURCE") would work

Reply:
oops, this ./ is correct